src/containers/Application.js

```diff
@@ -447,14 +445,17 @@ class Application extends Component {
 
 	addHelpButton(path) {
 		useEffect(() => {
-			this.HelpService.setData(path);
+			this.Store.dispatch({
+				type: SET_HELP_PATH,
+				path: path
```

Comment:
@aringocode Alright, sorry for requesting changes all over again :D Here the `path` should be renamed to `helpPath` or `helpContentPath` or something similar. `path` is just too general

Reply:
`helpPath: path`

Reply:
@Pe5h4 ok, i renamed it